packages/ui-shared/src/components/VideoBlock/VideoBlock.tsx

```diff
@@ -68,21 +70,49 @@ const VideoBlock: React.FC<IVideoBlockProps> = ({
     isMuted = true,
     isAutoplay = false,
     contentPositionRight = false,
+    poster,
 }) => {
     const renderVideo = React.useCallback(() => {
         switch (videoType) {
             case VideoTypes.YOUTUBE: {
-                const src = `https://www.youtube.com/embed/${videoSrc}?&autoplay=${isAutoplay ? 1 : 0}&mute=${
-                    isMuted ? 1 : 0
-                }&loop=1&rel=0&controls=0&showinfo=0e&iv_load_policy=3&playlist=${videoSrc}`;
+                const url = `https://www.youtube.com/embed/${videoSrc}?`;
+                const autoplay = `&autoplay=${isAutoplay ? 1 : 0}`;
+                const mute = `&mute=${isMuted ? 1 : 0}`;
+                const loop = `&loop=1`;
+                const rel = `&rel=0`;
+                const controls = `&controls=${isAutoplay ? 0 : 1}`;
+                const showinfo = `&showinfo=0e`;
```

Comment:
старая опечатка. по [доке](https://developers.google.com/youtube/player_parameters#showinfo) должно быть 0 | 1

```suggestion
                const showinfo = `&showinfo=0`;
```